utils/datetime/LocalDate.js

```diff
@@ -313,7 +313,7 @@ export class LocalDate {
             !m.isValid(),
             `Invalid argument for LocalDate: ${s}.  Use 'YYYYMMDD' or 'YYYY-MM-DD' format.`
         );
-        this._isoString = s;
+        this._isoString = s.slice(0, 4) + '-' + s.slice(4, 6) + '-' + s.slice(6, 8);
```

Comment:
seems like if we do this, we might as well cache the YYYY-MM-DD value,  and use that as the "main" format all around.  Basically need to reverse this login and add *in* the dashes in get()

This will work better if paired with a hypenated json serialization in Hoist Core as well.

Reply:
Also --- there should be a cleanup to the rendering of the cube label in Hoist Activity Monitor -- we were jumping through some hoops to get the dashes in there.

Reply:
Both suggestions applied in latest commits.